### PR TITLE
delete a grant by user-id

### PR DESCRIPTION
**Pull request by alireza-a**
This PR addresses https://github.com/go-auth0/auth0/issues/90.

I'm new to auth0 and I'm not sure how to consistently test this method.
Specially since the test cases run against an auth0 API and I can't find an easy way to generate grants!

### CODE REVIEW

**Comment by alexkappa** [comment]
Yeah thats a bit tricky. I am not entirely sure myself.

I’ll need to do some validation then manually and merge this in. If I can come up with a way to test it I’ll let you know.

Thanks!

**Comment by alireza-a** [comment]
👋 @alexkappa, I was just wondering if you had a chance to validate this PR?

File: management/grant.go [hunk 1]
```diff
@@ -32,3 +32,9 @@ func (m *GrantManager) List(opts ...ListOption) (g []*Grant, err error) {
 	err = m.get(m.uri("grants")+m.q(opts), &g)
 	return
 }
+
+// Delete revokes a grant associated with a user-id
+// https://auth0.com/docs/api/management/v2#!/Grants/delete_grants_by_id
+func (m *GrantManager) Delete(userID string, opts ...ListOption) error {
```
Comment (alexkappa) [inline management/grant.go]: From the documentation it seems that the argument here should be id, not userID. A user ID may be supplied using a parameter.

Example:

https://alexkappa.eu.auth0.com/api/v2/grants/123
https://alexkappa.eu.auth0.com/api/v2/grants/123?user_id=567

As per the documentation the grant id is a required field.

**Comment by alexkappa** [comment]
Hi @alireza-a, sorry for the late reply. One small issue and we should be good to go.

File: management/grant.go (hunk 1 above)
Comment (alireza-a) [reply management/grant.go]: fixed in [9d61ad3](https://github.com/go-auth0/auth0/pull/91/commits/9d61ad38e774c0810f1c56e5c26480b8cd457f01)

**Comment by alexkappa** [comment]
Thank you @alireza-a :)

**Comment by alireza-a** [comment]
@alexkappa would it be possible to create a new release with the latest changes on master?

**Comment by alexkappa** [comment]
Of course, here you go: [v3.3.1](https://github.com/go-auth0/auth0/releases/tag/v3.3.1)

**Pull request merged**